dspace/config/local.cfg.EXAMPLE

```diff
@@ -49,6 +49,9 @@ dspace.ui = xmlui
 # Name of the site
 dspace.name = DSpace at My University
 
+# assetstore.dir, look at DSPACE/config/spring/api/bitstore.xml for more options
+assetstore.dir = ${dspace.dir}/assetstore
```

Comment:
Tiny thing, but I'd suggest commenting this out in local.cfg.EXAMPLE by default. For most scenarios, the default setting is appropriate. I've tended to only *uncomment* things in local.cfg.EXAMPLE which are highly likely to be customized.